infrastructure/terraform/components/api/locals.tf

```diff
@@ -17,7 +18,8 @@ locals {
   common_lambda_env_vars = {
     LETTERS_TABLE_NAME = aws_dynamodb_table.letters.name,
     LETTER_TTL_HOURS = 24,
-    SUPPLIER_ID_HEADER = "nhsd-supplier-id"
-    APIM_CORRELATION_HEADER = "nhsd-correlation-id"
+    SUPPLIER_ID_HEADER = "nhsd-supplier-id",
+    APIM_CORRELATION_HEADER = "nhsd-correlation-id",
+    DOWNLOAD_URL_TTL_SECONDS = 3600
```

Comment:
1h seems too long. The design of the API means they should only need a link to be valid during the redirect, so 60 seconds should be enough.

Reply:
```suggestion
    DOWNLOAD_URL_TTL_SECONDS = 60
```